public/pyodideWorkerClean.js

```
@@ -0,0 +1,298 @@
+    const wantBcf = !!generateBcf
+
+    const now = new Date()
+    const yyyy = now.getFullYear()
+    const mm = String(now.getMonth() + 1).padStart(2, '0')
+    const dd = String(now.getDate()).padStart(2, '0')
+    const yy = String(yyyy).slice(-2)
+    const ifcBaseName = (fileName || 'report.ifc').toString().split(/[/\\]/).pop()
+    const idsBaseName = (idsFilename || 'ids').toString().split(/[/\\]/).pop()
+    const combinedName = `${yy}${mm}${dd}-${ifcBaseName}-${idsBaseName}`
+    const sanitizedReportBaseName = (combinedName || 'report')
+      .replace(/[<>:"/\\|?*\u0000-\u001F]/g, '_')
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)